### PR TITLE
sort audit list by time

### PR DESCRIPTION
**Pull request by fewdan**
sort audit list by time

### CODE REVIEW

File: pkg/utils/tui.go [hunk 1]
```diff
@@ -17,21 +17,37 @@ import (
...
+	return p[i][1] > p[j][1]
+}
+
+func (p table) Swap(i, j int) {
+	p[i], p[j] = p[j], p[i]
+}
+
 // PrintTable accepts a matrix of strings and print them as ASCII table to terminal
-func PrintTable(rows [][]string, header bool) {
+func PrintTable(rows table, header bool) {
```
Comment (lonng) [inline pkg/utils/tui.go]: Which may break other callers.

File: Makefile [hunk 2]
```diff
@@ -1,7 +1,9 @@
 GOVER := $(shell go version)
 
-GOOS    := $(if $(GOOS),$(GOOS),$(shell go env GOOS))
-GOARCH  := $(if $(GOARCH),$(GOARCH),amd64)
+#GOOS    := $(if $(GOOS),$(GOOS),$(shell go env GOOS))
```
Comment (lonng) [inline Makefile]: Why change this?

File: pkg/utils/tui.go (hunk 1 above)
Comment (fewdan) [reply pkg/utils/tui.go]: already edited

File: Makefile (hunk 2 above)
Comment (fewdan) [reply Makefile]: It was temporarily changed during the test and has been changed back.

**Pull request merged**